src/python/merge_tester.py

```diff
@@ -85,7 +85,7 @@ def write_cache(
         f.write(explanation)
 
 
-def read_cache(cache_file: str) -> Tuple[MERGE_STATES, float, str]:
+def read_cache_merge_status(cache_file: str) -> Tuple[MERGE_STATES, float, str]:
```

Comment:
This tuple is used several times.  Would it make sense to make it a datatype (a record) with named fields?

Reply:
I believe you would like to use NamedTuples, we could do that